cli/command/formatter/stats.go

```diff
@@ -37,11 +35,80 @@ type ContainerStatsAttrs struct {
 	PidsCurrent      uint64 // Not used on Windows
 }
 
-// ContainerStats represents the containers statistics data.
+// ClearContainerStats represents the statistics data collected from a container.
+type ClearContainerStats struct {
```

Comment:
Anyway, for the time being, I changed it to `ContainerStatsRecord`. Hope it is better than just `ClearContainerStats`.